static/app/views/issueDetails/streamline/header.tsx

```diff
@@ -76,245 +77,191 @@ export default function StreamlinedGroupHeader({
   }, [group, activeUser.id]);
 
   return (
-    <Header>
-      <StyledBreadcrumbs
-        crumbs={[
-          {
-            label: 'Issues',
-            to: {
-              pathname: `/organizations/${organization.slug}/issues/`,
-              query: query,
+    <Fragment>
+      <Header>
+        <Breadcrumbs
+          crumbs={[
+            {
+              label: 'Issues',
+              to: {
+                pathname: `/organizations/${organization.slug}/issues/`,
+                query,
+              },
             },
-          },
-          {label: shortIdBreadcrumb},
-        ]}
-      />
-      <HeadingGrid>
-        <Heading>
-          <TitleHeading>
-            <TitleWrapper>
-              <StyledEventOrGroupTitle data={group} />
-            </TitleWrapper>
-          </TitleHeading>
-          <MessageWrapper>
-            <EventMessage
-              data={group}
-              message={message}
-              type={group.type}
-              level={group.level}
-              showUnhandled={group.isUnhandled}
-              levelIndicatorSize={'10px'}
-            />
+            {
+              label: (
+                <ShortIdBreadcrumb
+                  organization={organization}
+                  project={project}
+                  group={group}
+                />
+              ),
+            },
+          ]}
+        />
+        <HeaderGrid>
+          <Flex gap={space(0.75)} align="flex-end">
```

Comment:
the secondary title doesn't look aligned with the primary title (too low)

Reply:
That's intentional from the designs: 
![image](https://github.com/user-attachments/assets/17fb675c-e22e-4eec-80fc-0d59e52631ba)

How it looks is another thing though 🤷 I don't mind it, since the primary title font is larger

Reply:
hmm , in the designs it seems like the bottom of each letter is aligned? but in the preview the secondary title looks like it's below